overlord/snapstate/flags.go

```diff
@@ -102,6 +102,9 @@ type Flags struct {
 	// QuotaGroupName represents the quota group a snap should be assigned
 	// to during installation.
 	QuotaGroupName string `json:"quota-group,omitempty"`
+
+	// Lane is the lane that tasks should join if Transaction is set to "all-snaps".
+	Lane int `json:"lane,omitempty"`
```

Comment:
do need to know this when running the tasks?

other we tend to clear such task creation only Flags in  Flags.ForSnapSetup ?

Reply:
No, just to add them to the lane. I'll clear it there. Thanks